rest/offer_types.php

```diff
@@ -40,7 +41,7 @@
 		echo json_encode($offer_type_arr);*/
 			} else {
 				$offer_type->create();
-				$offer_type->read();
+				//$offer_type->read();
```

Comment:
similarly we typically read after create to get those values

Reply:
This was also causing the errors `Trying to access array offset on value of type bool in ...`

Reply:
Again, we should look at why read is causing a problem

Reply:
I un-commented this and it didn't throw any errors so I don't think it is throwing an error anymore.